vizro-core/docs/pages/user_guides/custom_charts.md

```diff
@@ -136,13 +147,15 @@ The below examples shows a more involved use-case. We create and style a waterfa
                 ),
             ],
             controls=[
-                vm.Filter(column="x", selector=vm.Dropdown(title="Financial categories", multi=True)),
+                vm.Filter(column="x", selector=vm.Dropdown(title="Financial categories", multi=True)),# (1)!
             ],
         )
         dashboard = vm.Dashboard(pages=[page_0])
 
         Vizro().build(dashboard).run()
         ```
+
+        1.  Note how we are able to apply a filter even to a custom chart
```

Comment:
I would expect this as a basic functionality - using "even" kind of sounds like it's special? haha

```suggestion
        1.  Note how we are able to apply a filter to a custom chart
```